QA.md

```diff
@@ -0,0 +1,77 @@
+# Suggested Test Sequence
+
+1. **Simple Initialization**
+1. - [ ] Initialize a new wallet.
+1. - [ ] When creating a PIN, type the wrong PIN during the confirmation step.
+
+1. **Generate token error Tests**
+1. - [ ] Go to Settings and Create a new Token.
+1. - [ ] Enter token name Test Token, and click Next.
+1. - [ ] Enter token symbol TEST, and click Next.
+1. - [ ] Enter the amount of 100, and click Next.
+1. - [ ] Click on create token and check the error message.
+1. - [ ] Check that the token has not been created.
+
+1. **Receive Tests**
+1. - [ ] Go to the Receive Screen and check the QR Code.
+1. - [ ] Click on the address to copy it.
+1. - [ ] Click on New address and check that a new address was generated. The QRCode must update.
+1. - [ ] Click on Share and Cancel.
+1. - [ ] Click on Payment Request and generate a payment request of 10 HTR.
+1. - [ ] Pay the Payment Request using another wallet and check the confirmation message.
+
+1. **Send Tests**
+1. - [ ] Go to the Send Screen and check whether the camera loads correctly.
+1. - [ ] Click on Manual Info.
+1. - [ ] Type any random address and click Next. It must show an error message.
```

Comment:
"random text" instead of "any random address", because actual random addresses should not error, just likely burn funds

Reply:
Fixed. Thanks!